### PR TITLE
[RW-4718][risk=no] Attach required/optional/publicly displayed tags to registration pages

### PR DESCRIPTION
**Pull request by als364**
localhost_4200_login (1)
localhost_4200_login (2)
localhost_4200_login (3)
---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have run and tested this change locally

### CODE REVIEW

File: ui/src/app/pages/login/account-creation/account-creation-institution.tsx [hunk 1]
```diff
@@ -338,22 +335,27 @@ export class AccountCreationInstitution extends React.Component<Props, State> {
           <div style={{...styles.text, fontSize: 16, marginTop: '1rem'}}>
             Please complete Step 1 of 3
           </div>
-          <div style={{...styles.text, fontSize: 14, marginTop: '0.7rem'}}>
+          <div style={{
+            ...styles.text,
+            fontSize: 14,
+            marginTop: '0.7rem',
+            padding: '0.5rem',
```
Comment (NehaBroad) [inline ui/src/app/pages/login/account-creation/account-creation-institution.tsx:342]: nit: Can we extract it as a style object

File: ui/src/app/pages/login/account-creation/account-creation-institution.tsx [hunk 2]
```diff
@@ -401,12 +403,12 @@ export class AccountCreationInstitution extends React.Component<Props, State> {
...
             {this.displayEmailErrorMessageIfNeeded()}
             <div style={{marginTop: '.5rem'}}>
-              <label style={{...styles.boldText, marginTop: '1rem'}}>
-                Which of the following best describes your role?
-                <i style={{...styles.publiclyDisplayedText, marginLeft: '0.2rem'}}>
-                  Publicly displayed
-                </i>
-              </label>
+              <FlexRow style={{alignItems: 'center', margin: '.5rem 0'}}>
+                <label style={{...styles.boldText}}>
```
Comment (NehaBroad) [inline ui/src/app/pages/login/account-creation/account-creation-institution.tsx:407]: we an use it as styles.boldText

**Comment by NehaBroad** [comment]
While testing this i found two issues:

- Your research background Subheader still has Publicly displayed text
Screen Shot 2020-04-17 at 10 09 40 AM

- The bullet points text is not indent properly 
Screen Shot 2020-04-17 at 10 15 18 AM

**Comment by als364** [comment]
localhost_4200_login(Smallest Size) (1)
New screenshot.

File: ui/src/app/pages/login/account-creation/account-creation-institution.tsx [hunk 3]
```diff
@@ -338,22 +343,20 @@ export class AccountCreationInstitution extends React.Component<Props, State> {
...
           {!loadingInstitutions && <div style={{marginTop: '.5rem'}}>
-            <label style={styles.boldText}>
-              Select your institution
-              <i style={{...styles.publiclyDisplayedText, marginLeft: '0.2rem'}}>
-                Publicly displayed
-              </i>
-            </label>
+            <FlexRow style={{alignItems: 'center', margin: '.5rem 0'}}>
+              <label style={styles.boldText}>Select your institution</label>
+              <PubliclyDisplayed style={{marginLeft: '1rem'}}/>
```
Comment (NehaBroad) [inline ui/src/app/pages/login/account-creation/account-creation-institution.tsx:358]: super nit: can the style be like style={marginLeft: '1rem'}

File: ui/src/app/pages/login/account-creation/account-creation.tsx [hunk 4]
```diff
@@ -436,33 +439,38 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
...
-          <div style={{...styles.text, fontSize: 12, marginTop: '0.7rem'}}>
+          <div style={{...styles.text, fontSize: 12, marginTop: '0.3rem'}}>
             All fields required unless indicated as optional</div>
-          <Section header={<div>Create an <i>All of Us</i> username</div>}>
+          <Section header={<div>Create an <i>All of Us</i> username</div>} style={{marginTop: '1.25rem'}}>
             <div>
               <FlexRow>
                 <FlexColumn>
-                  <div style={{...styles.asideText, marginBottom: '0.3rem'}}>
+                  <div style={{...styles.asideText}}>
```
Comment (NehaBroad) [inline ui/src/app/pages/login/account-creation/account-creation.tsx:455]: nit: can style be style={styles.asideText}

File: ui/src/app/pages/login/account-creation/account-creation.tsx [hunk 5]
```diff
@@ -436,33 +439,38 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
...
+                      value={username}
+                      inputId='username'
+                      inputName='username'
+                      placeholder='New Username'
+                      invalid={this.state.usernameConflictError || this.usernameInvalidError()}
+                      containerStyle={{width: '26rem', marginTop: '1.25rem'}}
+                      labelText={usernameLabelText}
+                      onChange={v => this.usernameChanged(v)}
+                  >
+                    <div style={{...inputStyles.iconArea}}>
```
Comment (NehaBroad) [inline ui/src/app/pages/login/account-creation/account-creation.tsx:469]: nit: style={inputStyles.iconArea}?

File: ui/src/app/pages/login/account-creation/account-creation.tsx [hunk 6]
```diff
@@ -566,15 +585,41 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
...
+                      )}
+                  >
+                    <i>All of Us</i> participants are most interested in knowing:
+                  </div>
+                  <ClrIcon shape='angle' style={{
+                    transform: this.state.showMostInterestedInKnowingBlurb ? 'rotate(180deg)' : 'rotate(90deg)'
+                  }}/>
+                </FlexRow>
+                {this.state.showMostInterestedInKnowingBlurb &&
+                  <ul style={{...styles.asideList, listStylePosition: 'outside', marginLeft: '0.75rem'}}>
```
Comment (NehaBroad) [inline ui/src/app/pages/login/account-creation/account-creation.tsx:614]: can the listStylePositon move inside asideList ?

Comment (NehaBroad) [reply ui/src/app/pages/login/account-creation/account-creation.tsx:614]: Question as i have not tried this, do we need style asideList both at ul and li level?

**Comment by NehaBroad** [comment]
There is an issue with margin on Create account step 2 (where we enter first name and username details) lets talk offline about this

File: ui/src/app/pages/login/account-creation/account-creation.tsx (hunk 6 above)
Comment (als364) [reply ui/src/app/pages/login/account-creation/account-creation.tsx:614]: We don't - we are using `asideText` on the `li`s.

**Comment by als364** [comment]
That margin change is intentional - take a look at the mocks: https://projects.invisionapp.com/d/main#/console/18996352/397916423/preview

This actually reminds me I need to do that for the demographics survey too.

**Comment by NehaBroad** [comment]
And create account-institution page too

File: ui/src/app/pages/profile/demographic-survey.tsx [hunk 7]
```diff
@@ -112,21 +117,28 @@ export class DemographicSurvey extends React.Component<Props, State> {
...
-          Please complete Step {requireInstitutionalVerification ? '3 of 3' : '2 of 2'}
-        </div>
-        <div>
+        {this.props.showStepCount &&
+          <div style={{fontSize: 16, marginBottom: '.5rem'}}>
+            Please complete Step {requireInstitutionalVerification ? '3 of 3' : '2 of 2'}
+          </div>
+        }
+        <div style={{
+          backgroundColor: colorWithWhiteness(colors.accent, .85),
```
Comment (NehaBroad) [inline ui/src/app/pages/profile/demographic-survey.tsx:129]: More of a lou question and Not part of this story (we can discuss this in UI meeting) but should the background color be consistent with Publicly displayed icon i.e colors.accent with 0.75

Comment (als364) [reply ui/src/app/pages/profile/demographic-survey.tsx:129]: Yeah, it probably should given the background image issue

**Comment by als364** [comment]
localhost_4200_login (5)
localhost_4200_login (6)
localhost_4200_login (7)
Updated screenshots.

**Pull request merged**